### ISSUE TITLE
The Dom parser crashes on Polyline and Polygon

### ISSUE THREAD

**Issue by tatarize** [mathandy/svgpathtools#70]
The Document class uses a dict to lookup the functions for the leaf node objects and calls them from the svg_to_paths file but the polyline and polygon functions do not properly take a group they specifically take a points_string, so they actually fail when called like that.

>     """converts the string from a polyline points-attribute to a string for a
>     Path object d-attribute"""

Compare this to something like ellipse:

>     """converts the parameters from an ellipse or a circle to a string for a 
>     Path object d-attribute"""

Do note, that simply fixing the polyline function to set `polyline_d = polyline['d']` will seemingly change the functionality of dom2dict() in svg_to_paths:

```
    if convert_polygons_to_paths:
        pgons = [dom2dict(el) for el in doc.getElementsByTagName('polygon')]
        d_strings += [polygon2pathd(pg['points']) for pg in pgons]
        attribute_dictionary_list += pgons

```

I rewrote the functions in a consistent manner in the svg_parser.py file in pull request #69. Generally because it the inconsistency of extracting the points early for those two conversions bits was an eyesore.

**Comment by tatarize** [comment]
In the tests, throw this line:

>     <polyline points="183,183 0,30 0,0"/>
   
Anywhere in groups.svg and run the test, it will instantly crash:

> Error
> Traceback (most recent call last):
>   File "C:\Python27\lib\unittest\case.py", line 329, in run
>     testMethod()
>   File "...\test\test_groups.py", line 51, in test_group_flatten
>     result = doc.flatten_all_paths()
>   File "...\svgpathtools\document.py", line 219, in flatten_all_paths
>     path_filter, path_conversions)
>   File "...\svgpathtools\document.py", line 142, in flatten_all_paths
>     path = transform(parse_path(converter(path_elem)), path_tf)
>   File "...\svgpathtools\svg_to_paths.py", line 53, in polyline2pathd
>     points = COORD_PAIR_TMPLT.findall(polyline_d)
> TypeError: expected string or buffer

**Comment by tatarize** [comment]
This may have been corrected by #85.

**Comment by tatarize** [comment]
Corrected.
